src/main/java/mate/academy/spring/model/dto/response/ShoppingCartResponseDto.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.spring.model.dto.response;
+
+import java.util.List;
+
+public class ShoppingCartResponseDto {
+    private Long id;
+    private List<Long> ticketsId;
```

Comment:
Consider renaming the field 'ticketsId' to 'ticketIds' for better readability and to follow naming conventions.